stock_move_location/wizard/stock_move_location_line.py

```diff
@@ -92,6 +92,21 @@ def create_move_lines(self, picking, move):
             if not self.env.context.get("planned") and values.get("qty_done") <= 0:
                 continue
             self.env["stock.move.line"].create(values)
+            if self.env.context.get("planned"):
+                available_quantity = self.env["stock.quant"]._get_available_quantity(
+                    line.product_id,
+                    line.origin_location_id,
+                    lot_id=line.lot_id,
+                    strict=False,
+                )
+                # Create stock.move.line
+                move._update_reserved_quantity(
+                    values.get("qty_done"),
+                    available_quantity,
+                    line.origin_location_id,
+                    lot_id=line.lot_id,
+                    strict=False,
+                )
```

Comment:
It's a bit hard to understand the context of this additoon. Can you please try and find a way to write on created move lines, as that should keep records consistent (and we should be able to remove this part).